cernopendata/modules/fixtures/data/records/cms-tools-higgsexample20112012.json

```diff
@@ -112,7 +112,7 @@
i></ul>\n</ul>\n  <li><strong>Reproduce the full example analysis</strong> (up to ~1 month or more on single CPU with fast internet connection, depending on internet connection speed and computer performance)</li>\n <ul><li>start by running level 3 and understand what you have done</li>\n <li> download <a href=\"/record/5500/files/demoanalyzer_cfg_level4data.py\">demoanalyzer_cfg_level4data.py</a> and <a href=\"/record/5500/files/demoanalyzer_cfg_level4MC.py\">demoanalyzer_cfg_level4MC.py</a> from this record</li>\n <li>at this level, instead of running over a single file, you will run over so-called index files which contain chains of files</li>\n <li> download all the data index files for the datasets listed in <a href=\"/record/5500/files/List_indexfile.txt\">List_indexfile.txt</a> to the <code>datasets</code> directory (you can find the links to the datasets in this record)</li>\n <li> download <a href=\"/record/1001\">the 2011 JSON validation file</a> to the <code>datasets</code> directory (in which you should already have the 2012 one)</li>\n <li> download all the MC index files for the MC sets listed in <code>List_indexfile.txt</code> to the <code>MCsets</code> directory (after having created it)</li>\n <li> edit the relevant demoanalyzer_cfg file and insert the index file you want; for data, make sure to use the correct JSON validation file in each case; set an output file name of your choice for each sample which you will recognise</li>\n <li> run the analysis job (<code>cmsRun demoanalyzer_cfg_level4...</code>) sequentially on all the input samples listed in <code>List_indexfile.txt</code>, i.e. produce all root output files yourself. If you have access to a computer farm with local support for the installation of the CMS software (the Open Data team can only provide support for the single virtual machine mode), you may also run the analysis in parallel on different CPUs, correspondingly speeding up the result.</li>\n <li> merge all the files from different index files of a dataset by using root tools</li>\n>ul><li>If everything ran correctly, at this stage the content of your rootfiles should be identical to the one in the predefined rootfiles. If you don't want to run everything, you can also run only some of the datasets (but all events of all index files of a dataset must be treated and merged), and use the predefined ones for the others. </li> </ul>\n <li> go to 2., using your own root output files instead of the predefined ones, i.e. edit the file <a href=\"/record/5500/files/M4Lnormdatall.cc\">M4Lnormdatall.cc</a> accordingly. If everything worked correctly you should get an output identical to mass4lcombine.pdf, but now using your own files.</li>\n </ul></ol>\n In addition to these instructions, which guide you through the example in detail, <a href=\"https://github.com/cms-opendata-analyses/HiggsExample20112012\">a github repository</a> based on this original code with minor modifications for direct download is available.</p> "
```

Comment:
There is an unnecessary `ul>` in the end of the second to last bullet in 4. and perhaps for 5, you could change it to `<strong>Go to 2.</strong>` to match the rest

Reply:
Whoops, thanks, I should have seen that. No fixed to <ul> as intended